Additions/XCTestCase-KIFAdditions.m

```diff
@@ -34,7 +34,18 @@ - (void)failWithException:(NSException *)exception stopTest:(BOOL)stop
 {
     self.continueAfterFailure = YES;
 
-    [self recordFailureWithDescription:exception.description inFile:exception.userInfo[@"FilenameKey"] atLine:[exception.userInfo[@"LineNumberKey"] unsignedIntegerValue] expected:NO];
+    NSString *filePath = exception.userInfo[@"FilenameKey"];
```

Comment:
Would you mind wrapping this and below usages of `XCTIssue` in a `#ifdef __IPHONE_14_0` and put the original implementation in the `#else`? That way this is backwards compatible.